build/utils/theoReplaceAll.js

```diff
@@ -0,0 +1,2 @@
+module.exports = (str, find, replace) =>
+  str.replace(new RegExp(find, 'g'), replace);
```

Comment:
@eltongarbin 
Just a JS best practice doubt. 
Is it a best practice separate these "functions" one per file?

Reply:
I don't think so.
It's more personal preference ;)